docs/admin/scale/index.md

```diff
@@ -0,0 +1,36 @@
+We regularly scale-test Coder against various reference architectures. Additionally, we provide a [scale testing utility](#scaletest-utility) which can be used in your own environment to give insight on how Coder scales with your deployment's specific templates, images, etc.
```

Comment:
> We regularly scale-test Coder

As a customer, I'd like to reproduce Coder's results, but the doc doesn't mention the used release version. It might be good to keep the version and date of performing load tests.

Reply:
Totally agree about including versions and dates! GKE refers this as `ubuntu_containerd`, which varies based on the Kubernetes version, so using this as a mental note to include the cluster version too ;) 

<img width="618" alt="Screen Shot 2022-12-01 at 3 26 28 PM" src="https://user-images.githubusercontent.com/22407953/205180490-18b66af7-001c-4358-9357-1c1c7e74649f.png">